beacon-chain/sync/pending_attestations_queue_test.go

```diff
@@ -258,6 +263,165 @@ func TestProcessPendingAtts_HasBlockSaveUnAggregatedAttElectra(t *testing.T) {
 	cancel()
 }
 
+func TestProcessPendingAtts_HasBlockSaveUnAggregatedAttElectra_VerifySeen(t *testing.T) {
```

Comment:
I ran `TestProcessPendingAtts_HasBlockSaveUnAggregatedAttElectra_VerifySeen` with `s.setSeenCommitteeIndicesSlot(data.Slot, data.CommitteeIndex, att.GetAggregationBits())` the test still passed

Reply:
Good catch should update the values in the test